deploy/openshift/operator.yaml

```diff
@@ -47,6 +47,17 @@ spec:
           limits:
             memory: "4Gi"
             cpu: "1"
+        livenessProbe:
+          httpGet:
+            path: /metrics
```

Comment:
in DVO this is a very data heavy endpoint which I don't think is wise to query every 10s in addition to what Prometheus itself is doing. If we don't have a proper `/health` endpoint, I guess we'd rather start with a tcp probe.